.gitignore

```diff
@@ -1 +1,5 @@
 coverage.txt
+.idea/
```

Comment:
I am going to merge that anyway, but I believe it is a good idea to not include environment-specific ignores in .gitignore of OSS projects. There are a lot of IDEs, editors and etcs, cannot really account them all. 

Instead, put appropriate ignores for whatever you use in `$XDG_CONFIG_HOME/git/ignore` and have them effective for all projects you work on.

Reply:
Thank you for noticing that, sounds like a reasonable thing to do. Also, `.git/info/exclude` if a project-specific ignore needed.